src/index.ts

```diff
@@ -1473,6 +1473,7 @@ export interface Translations {
   we_are_unable_to_fulfill_your_request?: string
   at_this_time_due_to_suspected_bot_like_behavior_as_indicated_by_our?: string
   verification?: string
+  and?: string
```

Comment:
Typically single word translations like this are an anti-pattern because different languages conjoin things differently.

Reply:
@sethyates , I'm after this, where TOS and PP are links, with `"and"`sandwiched in-between.  Not sure a way around that.

![image](https://github.com/ketch-sdk/ketch-types/assets/34262057/74f0d9f2-1b21-400c-ac62-7d3659d636bb)

Reply:
You would have a translation that has those two template replacements in there.

Reply:
So, you're suggesting I have a single entry in the translations `.json`.   How would I know where to split the string in each and every language so i could wrap some substrings in `<Link/>` and others not?

I see you're off for the rest of today.  I'll ping you on Monday and you can show me your solution of how you'd do that in the code.

![image](https://github.com/ketch-sdk/ketch-types/assets/34262057/8fccf003-a355-4d7a-91e4-018418fbaf07)